akita/src/api/entity-store.ts

```diff
@@ -71,13 +71,13 @@ export class EntityStore<S extends EntityState<E>, E> extends Store<S> {
    * @example
    * this.store.add([Entity, Entity]);
    * this.store.add(Entity);
-   * this.store.add(Entity,{prepend: true});
+   * this.store.add(Entity, { prepend: true });
    */
-  add(entities: E[] | E,config?:{prepend: boolean}) {
+  add(entities: E[] | E, addConfig ?: { prepend: boolean }) {
```

Comment:
make the params a default of empty object and remove the additional check in the _add function.
`add(entities: E[] | E, params: AddParams = {})`